src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts

```diff
@@ -120,17 +120,17 @@ function getStackLabelPointsForDualAxis(stacks: any[]) {
     return flatten(
         // 'column0' is primary axis and 'column1' is secondary axis
         zip(...stacks.map((item: any, index: number) => values(item[`column${index}`])))
-    );
+    ).filter((point: any) => point);
 }
 
 function toggleStackedLabelsForDualAxis() {
     const { yAxis } = this;
 
-    const stackTotalGroups = yAxis.map((axis: any) => axis.stackTotalGroup);
+    const stackTotalGroups = yAxis.map((axis: any) => axis.stackTotalGroup).filter((group: any) => group);
```

Comment:
move it to something like `getStackTotalGroups`

Reply:
Fixed